src/site/content/en/blog/css-size-adjust/index.md

```diff
@@ -0,0 +1,397 @@
+support). It also demonstrates a few ways to correct and normalize font sizes
+for smoother user experience, consistent design systems and more predictable
+page layout. For example, when optimizing initial font rendering to prevent
+[cumulative layout shift](https://web.dev/cls/) (CLS). 
+
+Here's an interactive [demo](https://codepen.io/argyleink/pen/rNyMjxR) of the
+problem space. Try changing the typeface with the dropdown and observe: 
+1. The wildheight differences in the results.
+1. Visual jarring content shifts. 
+1. Moving interactive target areas (the dropdown jumps around!).
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'WNpzRKd'
+} %}
+
+{% Aside 'key-term' %} 
+Font family vs Typeface: A **typeface** is referred to by
+its family name plus its font face. `Helvetica Bold` is referring to the
+specific bold typeface while `Helvetica` is generically referring to the entire
+family of 8+. With CSS `@font-face` you'll be dealing with typefaces, even
+though to use them you need to write `font-family`. 
+{% endAside %}
+
+## How to scale fonts with `size-adjust`
+
+An introduction to the syntax:
+
+```css/2
+@font-face {
+  font-family: "Adjusted Typeface";
+  size-adjust: 150%;
+  src: url(some/path/to/typeface.woff2) format('woff2');
+}
+```
+
+1. Creates a custom typeface named `Adjusted Typeface`.
+1. Uses `size-adjust` to scale up each glyph to 150% their default size.
+1. Affects only the single imported typeface.
+
+Use the above custom typeface like this:
+
+```css
+h1 {
+  font-family: "Adjusted Typeface";
+}
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'yLMKgRx'
+} %}
+
+{% Aside 'warning' %} 
+If the second headline in the above demo is not larger than the
+first one, your browser does not support `size-adjust`. 
+{% endAside %}
+
+### Mitigating CLS with seamless font swapping
+In the following gif, watch the examples on the left and how there's a shift
+when the font is changed. This is just a small example with a single headline
+element and the issue is very noticeable.
+
+<figure class="w-figure">
+  {% Video
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/6xhatUhszn7M6pDFcVgV.mp4",
+    autoplay="true",
+    loop="true",
+    muted="true"
+  %}
+  <figcaption class="w-figure">
+    left has layout shift, right does not
+  </figcaption>
+</figure>
+
+To improve font rendering, a great technique is [font
+swapping](https://web.dev/font-display/), where you render a quick-loading
+system font to show the content first, and then swap that with a web font when
+the web font finishes loading. This gives you the best of both worlds: the
+content is visible as soon as possible, and you get a nicely styled page without
+sacrificing your user's time to content. The problem, however, is that sometimes
+when the web font loads, it shifts the entire page around since it presents at a
+slightly different box height size, even though the font is set to the same
+size.
```

Comment:
```suggestion
To improve font rendering, a great technique is [font
swapping](/font-display/). Render a quick-loading
system font to show the content first, then swap that with a web font when
the web font finishes loading. This gives you the best of both worlds: the
content is visible as soon as possible, and you get a nicely styled page without
sacrificing your user's time to content. The problem however, is that sometimes
when the web font loads, it shifts the entire page around since it presents at a
slightly different box height size, even though the font is set to the same
size.
```